Dockerfile

```diff
@@ -1,5 +1,9 @@
 # Build the manager binary
-FROM golang:1.20 as builder
+FROM registry.access.redhat.com/ubi8-minimal:latest as builder
+ARG GO_PLATFORM=amd64
+env PATH=$PATH:/usr/local/go/bin
+RUN microdnf install tar gzip
+RUN curl -L --output - "https://golang.org/dl/go1.20.10.linux-${GO_PLATFORM}.tar.gz" | tar -xz -C /usr/local/
```

Comment:
The Go version should be parameterized. Link it to the OnePipeline's config `go-version`

https://github.ibm.com/websphere/operators/blob/3f38eb6677e355cdc7a5320a0c2cb9690709301f/scripts/pipeline/containerize-stage.sh#L109